### ISSUE TITLE
Feature: Create Home page layout

### ISSUE THREAD

**Issue by StraykerPL** [StraykerPL/Binder#36]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

- Add `Default Layout` component to main page's component,
- Add `Navbar` and `MainSpace` components to main page's component - both components are OUT OF SCOPE of this ticket,

**Describe alternatives you've considered**

**Additional context**

**Comment by StraykerPL** [comment]
Blocked by #56

**Comment by StraykerPL** [comment]
Rejected - there's no need for this component, Home component can serve like this.
